core/src/main/java/org/elasticsearch/action/bulk/BulkShardRequest.java

```diff
@@ -93,4 +91,9 @@ public void readFrom(StreamInput in) throws IOException {
         }
         refresh = in.readBoolean();
     }
+
+    @Override
+    public String toString() {
+        return "shard bulk {" + shardId() + "}";
```

Comment:
fixed in https://github.com/areek/elasticsearch/commit/6fa8140adf3b9c774431b5f815543930d144cde2